functions/yum/repo/baseurl_string.pp

```diff
@@ -0,0 +1,14 @@
+function simp::yum::repo::baseurl_string(
+  Array[Simp::Hostorurl] $servers,
+  String $simp_baseurl_path,
+) >> String {
```

Comment:
Make sure this works with Puppet 4.7. It does work with 4.9 but I had issues with it in 4.7